.rustfmt.toml

```diff
@@ -0,0 +1,2 @@
+reorder_imports = true
+max_width = 60
```

Comment:
Why do you specify a `max_width` of 60 ? I had this feeling that it is less possible to write function calls on one line and make things a little less easy to read.

Reply:
good catch, I just copied it from a project and also noticed I didn't like how it was reformatting things